robottelo/cli/base.py

```diff
@@ -17,30 +17,30 @@ class CLIBaseError(Exception):
     """Indicates that a CLI command has finished with return code different
     from zero.
 
-    :param return_code: CLI command return code
+    :param status: CLI command return code
     :param stderr: contents of the ``stderr``
     :param msg: explanation of the error
 
     """
 
-    def __init__(self, return_code, stderr, msg):
-        self.return_code = return_code
+    def __init__(self, status, stderr, msg):
+        self.status = status
```

Comment:
I prefer aligning this attribute with the variable name for the ssh status that's now consistent across the framework with this PR. 

If we want to separately create a `success` property on the class to represent boolean pass/fail of a given command (even if it just compares `self.status == 0`, I'd back that.

Reply:
status is more correct based on what the libssh2 API spec defines.

Reply:
Agree with @JacobCallahan and @mshriver but `status` is a little bit confusing, status leads to the pass or fail but the status code gives us some informative integer value, and based on that we can identify the problem, if we create a property of success then it would only return the `pass` and `fail` string based on their status code(0 and non 0) and that would be less informative. here we do not have such scenarios so I think the property would work but it needs a lot of changes and that makes no sense.